### PR TITLE
Storing data in DB and display data from DB

### PR DESCRIPTION
**Pull request by LisKorzun**

### CODE REVIEW

File: app/DoctrineMigrations/Version20150331004642.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?php
```
Comment (EduardTrayan) [inline app/DoctrineMigrations/Version20150331004642.php]: This file is redundant, please remove it

File: src/TestLab/WebsiteBundle/Entity/Contact.php [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+     *
+     * @ORM\Column(name="body", type="text", length=1000)
+     * @Assert\NotBlank(message = "Body is required")
+     * @Assert\Length(
+     *      min = 3,
+     *      max = 1000
+     * )
+     */
+    private $body;
+
```
Comment (EduardTrayan) [inline src/TestLab/WebsiteBundle/Entity/Contact.php]: Please remove blank line

File: src/TestLab/CommonBundle/Controller/AbstractController.php [hunk 3]
```diff
@@ -23,6 +24,10 @@
     private $formFactory;
 
     /**
+     * @var ContactRepository
+     */
+    private $contactRepository;
```
Comment (EduardTrayan) [inline src/TestLab/CommonBundle/Controller/AbstractController.php]: This repository isn't supposed to be a common dependencies to all your controllers, so move it to target controller

File: app/DoctrineMigrations/Version20150331004642.php (hunk 1 above)
Comment (LisKorzun) [inline app/DoctrineMigrations/Version20150331004642.php]: ок

File: src/TestLab/WebsiteBundle/Entity/Contact.php (hunk 2 above)
Comment (LisKorzun) [inline src/TestLab/WebsiteBundle/Entity/Contact.php]: ок

**Pull request merged**